doc/radosgw/multisite.rst

```diff
@@ -945,100 +953,106 @@ the master zonegroup, specify the ``--master`` flag. For example:
 .. note:: Use ``zonegroup modify --rgw-zonegroup=<zonegroup-name>`` to modify
           an existing zone group’s settings.
 
-Make a Zone Group the Default
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Making a Zonegroup the Default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-One zonegroup in the list of zonegroups should be the default zonegroup.
-There may be only one default zonegroup. If there is only one zonegroup
-and it wasn’t specified as the default zonegroup when it was created,
-make it the default zonegroup. Alternatively, to change which zonegroup
-is the default, execute:
+One zonegroup in the list of zonegroups must be the default zonegroup.  There
+can be only one default zonegroup. In the case that there is only one zonegroup
+which was not designated the default zonegroup when it was created, use the
+folloiwng command to make it the default zonegroup. Commands of this form can
+be used to change which zonegroup is the default. 
 
-.. prompt:: bash #
+#. Designate a zonegroup as the default zonegroup:
 
-   radosgw-admin zonegroup default --rgw-zonegroup=comedy
+   .. prompt:: bash #
 
-.. note:: When the zonegroup is default, the command line assumes
-          ``--rgw-zonegroup=<zonegroup-name>`` as an argument.
+      radosgw-admin zonegroup default --rgw-zonegroup=comedy
 
-Then, update the period:
+   .. note:: When the zonegroup is default, the command line assumes that the
+   name of the zonegroup will be the argument of the
+   ``--rgw-zonegroup=<zonegroup-name>`` option. (In this example,
+   ``<zonegroup-name>`` has been retained for the sake of consistency and
+   legibility.)
```

Comment:
note the sphinx warning here

Reply:
Good catch, @cbodley. This is a quirk of Sphinx. Admonitions that appear within ordered lists cannot be indented like this, but must run on in one giant line. As far as I can tell, that isn't documented anywhere except in my own hand-written, inked notes, and in this comment.